browser/src/control/Control.ServerAuditDialog.ts

```diff
@@ -120,6 +167,66 @@ class ServerAuditDialog {
 		);
 	}
 
+	private showClipboardSnackbar(message: string, color: string): void {
+		const snackbar = document.createElement('div');
+		snackbar.textContent = '';
+		snackbar.textContent = message;
+		snackbar.style.position = 'fixed';
+		snackbar.style.bottom = '20px';
+		snackbar.style.left = '50%';
+		snackbar.style.transform = 'translateX(-50%)';
+		snackbar.style.backgroundColor = '#4CAF50';
+		snackbar.style.color = 'white';
+		snackbar.style.padding = '12px 24px';
+		snackbar.style.borderRadius = '4px';
+		snackbar.style.boxShadow = '0 2px 6px rgba(0, 0, 0, 0.3)';
+		snackbar.style.zIndex = '9999';
+
+		document.body.appendChild(snackbar);
+
+		setTimeout(() => {
+			snackbar.remove();
+		}, 3000); // hide after 3 seconds
+	}
+
+	private additionalWarnings: any[] = [];
+
+	private copyAuditToClipboard(text: string): void {
+		if (this.isClipboardAvailable()) {
+			navigator.clipboard
```

Comment:
We have clipboard handling done already.
Please do not interact with it directly, let's follow the same pattern as in the About dialog:

https://github.com/CollaboraOnline/online/blob/0edb0befe43d200ff471ed5f813b86fe538f2f74/browser/src/control/Control.AboutDialog.ts#L274

put into input field and do our copy command